package.json

```diff
@@ -33,6 +33,8 @@
     "docs:watch": "npm run docs:prepare && gitbook serve",
     "docs:publish": "npm run docs:clean && npm run docs:build && ./scripts/publish-docs",
     "flow": "eslint packages",
+    "todo": "leasot 'packages/**/*.js' --ignore '**/node_modules','**/dist','**/lib'",
```

Comment:
I am confused by the integration. Don't you want to do this in lint-staged on any js file change and just auto git add things?

Reply:
Ahh, good question. I want to run through the whole repo in order to build a correct `TODO.md`. So running it on any js file change inside `lint-staged.config.js` would run the script multiple times, no?

Reply:
Nope. It runs it given that a given set of currently staged files matching you glob. Ref: https://github.com/okonet/lint-staged#packagejson-example

Reply:
If you want to run it against anything then a normal pre-commit hook would do. Husky is very nice for that. https://github.com/typicode/husky

Reply:
I think generally 

1. You need to run it only once for all files
2. Then have a hook for js files with lint-staged

Reply:
Im not following what the second step would do here?

Anyways, I shortened the scripts so they make more sense here https://github.com/commercetools/nodejs/pull/670/commits/bd2ad8a9190b16f55720720c42f7429596f78a35

So basically, `npm run todo` will just log out all errors to the terminal.
`todo-staged` is being run in the Husky `precommit` hook after `lint-staged` in order to create the `.md` with all the todo's we currently have.